packages/react-ui-core/src/Ratings/LinearGradient.js

```diff
@@ -3,9 +3,10 @@ import PropTypes from 'prop-types'
 
 export default class LinearGradient extends Component {
   static propTypes = {
-    id: PropTypes.string,
-    color: PropTypes.string,
-    width: PropTypes.string,
+    id: PropTypes.string.isRequired,
+    fillColor: PropTypes.string.isRequired,
```

Comment:
getting some warnings about these in other places `LinearGradient` is being used